### PR TITLE
SCA: dropped import duplicates, fixed a few cases of php core API misuse

### PR DESCRIPTION
**Pull request by kalessil**

### CODE REVIEW

**Comment by Naktibalda** [comment]
Thanks

**Comment by kalessil** [comment]
You are welcome =) If changes looks good, and you folks are using PhpStorm, I'd supply maintainers with 2-3 Php Inspections (EA Ultimate) (https://plugins.jetbrains.com/plugin/10215-php-inspections-ea-ultimate-) 1-year licenses.

File: src/Codeception/Module/WebDriver.php [hunk 1]
```diff
@@ -798,25 +798,15 @@ public function resizeWindow($width, $height)
     public function seeCookie($cookie, array $params = [])
     {
         $cookies = $this->filterCookies($this->webDriver->manage()->getCookies(), $params);
-        $cookies = array_map(
-            function ($c) {
-                return $c['name'];
-            },
-            $cookies
-        );
+        $cookies = array_column($cookies, 'name');
```
Comment (Naktibalda) [inline src/Codeception/Module/WebDriver.php]: I don't know why, but 4 WebDriver tests calling `seeCookie` method fail on Semaphore:

```
There were 4 failures:

---------
1) WebDriverTest: Save session snapshots exclude invalid cookie domains
 Test  tests/web/WebDriverTest.php:testSaveSessionSnapshotsExcludeInvalidCookieDomains
Failed asserting that an array contains 'PHPSESSID'.
#1  /home/runner/Codeception/src/Codeception/Module/WebDriver.php:803
#2  /home/runner/Codeception/tests/web/WebDriverTest.php:718
#3  /home/runner/Codeception/codecept:43

---------
2) WebDriverTest: Cookies with path
 Test  tests/web/WebDriverTest.php:testCookiesWithPath
Failed asserting that an array contains 'cookie'.
#1  /home/runner/Codeception/src/Codeception/Module/WebDriver.php:803
#2  /home/runner/Codeception/tests/unit/Codeception/Module/TestsForWeb.php:813
#3  /home/runner/Codeception/tests/web/WebDriverTest.php:791
#4  /home/runner/Codeception/codecept:43

---------
3) WebDriverTest: Cookies
 Test  tests/web/WebDriverTest.php:testCookies
Failed asserting that an array contains 'test_cookie'.
#1  /home/runner/Codeception/src/Codeception/Module/WebDriver.php:803
#2  /home/runner/Codeception/tests/unit/Codeception/Module/TestsForWeb.php:796
#3  /home/runner/Codeception/tests/web/WebDriverTest.php:777
#4  /home/runner/Codeception/codecept:43

---------
4) WebDriverTest: Session snapshots
 Test  tests/web/WebDriverTest.php:testSessionSnapshots
Failed asserting that an array contains 'PHPSESSID'.
#1  /home/runner/Codeception/src/Codeception/Module/WebDriver.php:803
#2  /home/runner/Codeception/tests/web/WebDriverTest.php:655
#3  /home/runner/Codeception/codecept:43

```

https://semaphoreci.com/codeception/codeception/branches/pull-request-5639/builds/3

Comment (kalessil) [reply src/Codeception/Module/WebDriver.php]: Reverted 'array_column(...)' changes to see if the changes is causing the build to fail there.

**Comment by Naktibalda** [comment]
Cookies are not arrays, they are objects implementing ArrayObject interface.

**Comment by DavertMik** [comment]
Wow, thanks @kalessil I follow your project for a long time, you are doing a great job!

Personally, I do not write PHP a lot these days but thanks for the proposal. Maybe @Naktibalda you are interested in new PHP inspections?

**Pull request merged**